kuma/javascript/src/header/language-menu.jsx

```diff
@@ -28,11 +28,18 @@ export default function LanguageMenu({ document }: Props): React.Node {
     // and we can just use the document language string without translation.
     let label =
         document.locale === 'en-US' ? gettext('English') : document.language;
+    let chooseLanguageString = gettext('Choose your preferred language.');
+    let currentLangPre = gettext('Current language is');
 
     return (
-        <Dropdown label={label} right={true}>
+        <Dropdown
+            label={label}
+            right={true}
```

Comment:
Can't this just be `right` without the `right={true}`?

Reply:
Thank you for your comment here @joedarc. Not sure if we should change this in this PR as it is:

1. Unrelated to the work here
2. Affects more components than the lang selector

Perhaps file an issue for this. It seems like a great `good-first-issue` to me. 🎉